esp/platform/espcache.cpp

```diff
@@ -296,9 +296,12 @@ void ESPMemCached::assertPool()
 extern esp_http_decl IEspCache* createESPCache(const char* setting)
 {
 #ifdef USE_LIBMEMCACHED
-    Owned<ESPMemCached> espCache = new ESPMemCached();
+    //An array of ESPMemCached is stored in MapStringToMyClass which does not support owned.
```

Comment:
It looks to me like it does support owned.  Even if it didn't I would suggest using owned in the code outside of the map for leak safety.  But I think it does support owned so hopefully you can completely use owned behavior.  Just test and make sure your objects are getting destroyed when they are done.